governance-exercises/Le Bateleur - Interpretation Exercise 01/Assessment of Ambiguity.md

```diff
@@ -0,0 +1,20 @@
+# Interpretation Exercise 01
+
+---
+
+## Is there ambiguity in these clauses?
+
+I assess these clauses as ambiguous/overbroad. In making this assessment, I am guided by analogous principles of statutory interpretation whereby ambiguity/overbreadth is determined not only by referencing the language at issue, but also the specific context in which that language appears and the broader context of the statutory scheme as a whole.  
+
+The following language is problematic:
+
+- "...when this relates to unintended consequences or mistakes in the affected documents." (GOV 12.1)
+- "In case of unintended consequences or mistakes in the Atlas and Scope Artifacts causing Maker Governance to function incorrectly..." (GOV 12.1.1)
+
+The most glaring deficiency is the phrase "when this relates to". This is an [omnibus term](https://www.law.cornell.edu/wex/omnibus_clause#:~:text=A%20contractual%20provision%20that%20operates,a%20last%20will%20and%20testament.) - an indeterminate catch-all (i.e., "relating to", pertaining to", "concerning") common to legal artifacts. Its purpose is to sweep under the legal instrument's purview events and conditions not actually specified in the instrument. This phrase enables an expansive reading of GOV 12 that poses a significant risk to Maker Governance.
+
+Several governance provisions - such as the "bootstrapping" elements of which GOV 12 is one - are framed loosely to encompass contingencies the drafter cannot anticipate. While this may be advisable from a limited perspective, it is decidedly risky when granting a single class of actors (here, the Governance Facilitators) too wide a discretion to supersede customary governance protocols. 
+
+When drafting a bootstrapping provision such as GOV 12, one must provide a reasonable opportunity, i.e., fair notice, to discern what is prohibited and what is allowed. The clauses here are problematic because their application to a specific subject or purpose is not sufficiently restricted. A huge variety of agendas can theoretically be swept under the clauses' general and indefinite characterization. It would not be onerous for misaligned actors to posit *some relation* of their agenda to "unintended consequences or mistakes" in the rulebooks "causing Maker Governance to function incorrectly." Without more explicit and narrowly tailored standards of application, GOV 12 runs the risk of encouraging arbitrary, biased enforcement where Governance Facilitators adjudicate GOV 12 matters on an ad hoc basic. 
```

Comment:
Yes, the ad-hoc nature thing is definitely something that has happened in the past, and generally hasn't led to great outcomes, imo. Agree that being more restricted can help avoid this issue.